content/about/feature-stages/index.md

```diff
@@ -90,7 +94,6 @@ Below is our list of existing features and their current phases. This informatio
 | [Kubernetes: Envoy Installation and Traffic Interception](/docs/setup/kubernetes/)        | Stable
 | [Kubernetes: Istio Control Plane Installation](/docs/setup/kubernetes/) | Stable
 | [Attribute Expression Language](/docs/reference/config/policy-and-telemetry/expression-language/)        | Stable
-| [Mixer In-Process Adapter Authoring Model](/blog/2017/adapter-model/)        | Deprecated
```

Comment:
instead of saying Deprecated, would it make sense to state `Deprecated and removed` so operators know a deprecated feature is no longer available?  In 1.3, we could move the line entirely.

Reply:
Happy to comply.  @douglas-reid thoughts?

Reply:
Happy to comply, @douglas-reid thoughts?

Reply:
I don't think we need to keep it. It was `Deprecated` one release ago. No new in-process adapters have been accepted. And we've been actively rejecting them for (most of) 2 release cycles. I think it is time to just remove it.

Reply:
agreed, we should just remove it

Reply:
removed